ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -370,3 +384,57 @@ def check_obc_option(self, text="Object Bucket Claims"):
         namespace_obj = OCP(kind=constants.NAMESPACE, namespace=sc_name)
         namespaces.append(namespace_obj)
         delete_projects(namespaces)
+
+
+class ObUi(BucketsUi):
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    def delete_ob_ui(self, delete_via, expect_fail):
+        """
+        Delete an OBC via the UI
```

Comment:
You are deleting an OB, please update the doc string accordingly

Reply:
fixed